drivers/ethernet/eth_stm32_hal.c

```diff
@@ -485,6 +498,19 @@ static void eth_isr(void *arg)
 	HAL_ETH_IRQHandler(heth);
 }
 
+void HAL_ETH_TxCpltCallback(ETH_HandleTypeDef *heth_handle)
```

Comment:
Are you sure, the F4 and F7 series driver have same function name in it's HAL_ETH lib? Anyway we'll se it, after build kite were run.

Reply:
F4 and F7 both have this TX complete interrupt, buildkite won't fail with this function declared. Otherwise, I can just enclose it with the defines.